lib/page-object/platforms/selenium_webdriver/page_object.rb

```diff
@@ -576,6 +576,31 @@ def cells_for(identifier)
         end
 
         #
+        # platform method to retrieve the text from a table row
+        # See PageObject::Accessors#tr
+        #
+        def tr_text_for(identifier)
```

Comment:
As above the corresponding td methods are `cell_text_for`, `cell_for`, and `cells_for`, so these should probably be `row_text_for`, `row_for`, and `rows_for`